tests/dogfood.spec.js

```diff
@@ -1,15 +1,32 @@
+//  'afterAll',
```

Comment:
Is this just to get something above the `beforeAll`?

Reply:
Also... Mmm, delicious dog food.

Reply:
Nope, that was an accident. I’ll take it out when I get to my computer next